src/components/PermissionPopup/PermissionPopup.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { View, Text, } from 'react-native'
+import { TouchableButton, } from '../index'
+import styles from './style'
+
+const PermissionPopup = ({ isWord, resolve, refresh, }) => {
+  const { container, popupBlock, headline, bottomsBlock, btn, btnText, } = styles
```

Comment:
Styles destructuring may be confusing

Reply:
Maybe destructuring is better for performance

Reply:
yeah, desctructuring is good enough to use it but not for styles